include/glob_opts.h

```diff
@@ -155,11 +135,15 @@ typedef float c_float;  /* for numerical values  */
 #   define c_print Rprintf
 #  else  /* ifdef MATLAB */
 #   define c_print printf
-#  endif /* ifdef MATLAB */
-
-/* Print error macro */
-#  define c_eprint(...) c_print("ERROR in %s: ", __FUNCTION__); c_print(\
-    __VA_ARGS__); c_print("\n");
+#  endif /* c_print configuration */
+
+/* error printing function */
+#  ifdef R_LANG
+#   define c_eprint Rprintf
```

Comment:
I would add a small comment on why it is defined differently than other languages (easier for us to remember later).